src/app/modules/submission/multirespondent-submission/multirespondent-submission.middleware.ts

```diff
@@ -126,6 +126,7 @@ const getSnapshottedFormDef = (
   form_fields: mrfSubmission.form_fields,
   form_logics: mrfSubmission.form_logics,
   workflow: mrfSubmission.workflow,
+  webhook: currentFormDef.webhook,
```

Comment:
the webhook URL is always loaded from the current form definition, along with these other fields (at time of writing):
- title
- respondent copy setting
- emails
- email notification
- step(s) to send notifications for